_test_/server.test.js

```diff
@@ -1,54 +1,78 @@
-const request = require('supertest');
 const app = require('../server/app');
+const supertest = require('supertest');
+const request = supertest(app);
+
 const mongoose = require('mongoose');
+mongoose.promise = global.Promise;
```

Comment:
My understanding is that with Mongoose 5 this is no longer necessary. I don't believe it'll hurt anything, but you may want to check out https://stackoverflow.com/q/51862570 for details.

Reply:
Yep, you are right. I removed this line and everything still works. I put this here just because it's from some of our old sprint. Didn't really dig into why I need this. Thanks for letting me know